multi-int-store/src/main/resources/application.yml

```diff
@@ -8,7 +8,6 @@ management:
     web:
       exposure:
         include: "*"
```

Comment:
I wonder if we should include all the endpoints or maybe just httptrace, health and loggers.

Reply:
I don't know which endpoints will prove useful yet. I would like to leave them open and play around with them a bit.